eng/common/tools.ps1

```diff
@@ -598,7 +598,7 @@ function MSBuild() {
 }
 
 #
-# Executes msbuild (or 'dotnet msbuild') with arguments passed to the function.
+# Executes msbuild (or 'dotnet build') with arguments passed to the function.
```

Comment:
changes in eng/common should probably be reverted and done in arcade

Reply:
thanks